client/layout/guided-tours/config-elements.js

```diff
@@ -188,7 +218,8 @@ export class Quit extends Component {
 	}
 
 	onClick = () => {
-		this.context.quit();
+		const { tour, tourVersion, step, isLastStep } = this.context;
+		this.context.next( { tour, tourVersion, step, isLastStep } );
```

Comment:
Looks like possibly copy and paste? Does @ehg remember? 😱

Also cf. w/ `master` where it's currently this:

```
    onClick = ( event ) => {
		this.props.onClick && this.props.onClick( event );
		this.context.quit();
	}
```